src/api/n2yo/index.ts

```diff
@@ -0,0 +1,15 @@
+import fetch from 'node-fetch'
+
+export default class N2YO {
+
+    static readonly apiAddress: string = 'https://api.n2yo.com/rest/v1/satellite'
+
+    static getTwoLineElements(id: string) {
+        const endPoint: string = `${this.apiAddress}/tle/${id}&apiKey=${process.env.N2YO_KEY}`
+        
+        return fetch(endPoint)
+            .then(res => res.json())
+            .catch(err => console.warn(err))
```

Comment:
You should send response upon catch, even if it's an error response. Otherwise HTTP request will stuck pending on browser side.